src/pages/AppLayout/AppLayout.tsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router-dom';
+import Container from 'src/components/core/Container/Container';
+
+const AppLayout = () => {
```

Comment:
I wouldn't name this `AppLayout` but `Layout` because we know it is for whole App